Mathlib/Algebra/Group/Equiv/Basic.lean

```diff
@@ -30,6 +30,26 @@ open Function
 
 variable {F α β M N P G H : Type*}
 
+namespace EquivLike
+
+@[to_additive (attr := simp)]
+theorem map_eq_one_iff {M N} [One M] [One N] [EquivLike F M N] [OneHomClass F M N]
```

Comment:
Should we use `simp` on `_root_.map_eq_one_iff`? Also, should this be generalized to `EmbeddingLike`?